game/notes.cc

```diff
@@ -44,6 +45,17 @@ double Note::scoreMultiplier() const {
 double Note::powerFactor(double note) const {
 	if (type == FREESTYLE) return 1.0;
 	double error = std::abs(diff(note));
+	switch(config["game/difficulty"].i()){
+		case 0: // normal mode (original)
+			//no points if error> 150 cents, perfect score if error <50 cents
```

Comment:
space between `error> 150 cents` -> `error > 150 cents` **and** `error <50 cents` -> `error < 50 cents`